src/quequations.jl

```diff
@@ -1,61 +1,158 @@
-    nb = size(coeffs(h), 1)
+@doc """
+Lindblad Master Equation type
+
+Fields :
+
+`hamiltonain` : Hamiltonian of the system
+`collapse_ops`  :  Collapse operators
+""" ->
+immutable QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix} <: QuEquation
+    hamiltonian::H
+    collapse_ops::Vector{V}
+    QuLindbladMasterEq(hamiltonian, collapse_ops) = new(hamiltonian, collapse_ops)
+end
+
+@doc """
+Lindblad Master Equation method
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonain of the system
+`collapse_ops` : Collapse operators
+""" ->
+QuLindbladMasterEq{H<:QuBase.AbstractQuMatrix, V<:QuBase.AbstractQuMatrix}(hamiltonian::H, collapse_ops::Vector{V}) = QuLindbladMasterEq{H,V}(hamiltonian, collapse_ops)
+
+@doc """
+Lindblad operator construct from the `Hamiltonian` and `collapse operators`
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonain of the system
+`collapse_ops` : Collapse operators
+""" ->
+function lindblad_op(hamiltonian::QuBase.AbstractQuMatrix, collapse_ops::Vector)
+    nb = size(coeffs(hamiltonian), 1)
+    nlop = length(collapse_ops)
+    heff = zeros(typeof(im*one(eltype(hamiltonian))), size(coeffs(hamiltonian)))
+    for l=1:nlop
+        heff = heff + 0.5*coeffs(collapse_ops[l])'*coeffs(collapse_ops[l])
+    end
     SI = Array(Int,0)
     SJ = Array(Int,0)
-    Lvals = Array(typeof(im*one(eltype(h))),0)
+    Lvals = Array(typeof(im*one(eltype(hamiltonian))),0)
     for m=1:nb
         for n=1:nb
             for i=1:nb
                 for j=1:nb
                     sm = (n-1)*nb + m
                     sj = (j-1)*nb + i
-                    lv = zero(Complex128)
+                    lv = zero(eltype(Lvals))
+                    for l=1:nlop
+                        lv = lv + collapse_ops[l][m,i]*collapse_ops[l]'[j,n]
+                    end
                     if j==n
-                        lv = lv - im*h[m,i]
+                        lv = lv - im*hamiltonian[m,i] - heff[m,i]
                     end
                     if i==m
-                        lv = lv + im*h[j,n]
+                        lv = lv + im*hamiltonian[j,n] - heff[j,n]
                     end
                     if real(lv)!=0 || imag(lv)!=0
                         push!(SI, sm)
                         push!(SJ, sj)
-                        push!(Lvals, lv)
+                        push!(Lvals, im*lv)
                     end
                 end
             end
         end
     end
-    return QuBase.similar_type(h)(sparse(SI, SJ, Lvals, nb*nb, nb*nb))
+    return QuBase.similar_type(hamiltonian)(sparse(SI, SJ, Lvals, nb*nb, nb*nb))
 end
 
+@doc """
+Liouvillian operator construct from the `Hamiltonian` and passing an empty array to `lindblad_op`
+
+Input Parameters :
+
+`hamiltonian` : Hamiltonain of the system
+""" ->
+liouvillian_op(hamiltonian::QuBase.AbstractQuMatrix) = lindblad_op(hamiltonian, [])
+
 function liouvillian_tensor(h::QuBase.AbstractQuMatrix)
     return -im*(tensor(eye(h), h) - tensor(h',eye(h)))
```

Comment:
Although we don't use it one should make it consistent with the output of `liouvillian_op` by removing the `-im`. This function also requires some documentation or at least a comment what it is good for.